### PR TITLE
fix(replay): Force playback speed of replay clips to 1x

### LINKED ISSUE #65007: [Replay Clip] Replay playback speed comes from local storage, but there's no way to change it in the clip UI
This dropdown isn't present:
SCR-20240212-hrax

### PR DESCRIPTION
**Pull request by ryan953**
Fixes https://github.com/getsentry/sentry/issues/65007

### CODE REVIEW

File: static/app/components/events/eventReplay/replayClipPreview.tsx [hunk 1]
```diff
@@ -207,9 +208,10 @@ function ReplayClipPreview({
 
   return (
     <ReplayContextProvider
+      analyticsContext={analyticsContext}
       isFetching={fetching}
+      prefsStrategy={StaticReplayPreferences()}
       replay={replay}
```
Comment (ryan953) [inline static/app/components/events/eventReplay/replayClipPreview.tsx:214]: @malwilley i noticed that we are not setting `initialTimeOffsetMs` here, should we make it so the replay starts at the time of the error?

**Review by c298lee — APPROVED** [review]

**Comment by c298lee** [comment]
I've noticed that skip inactivity applies even when I have it unchecked. It's currently happening for a while tho, I just noticed it again when testing, so it's not from this PR.

**Comment by ryan953** [comment]
> I've noticed that skip inactivity applies even when I have it unchecked. It's currently happening for a while tho, I just noticed it again when testing, so it's not from this PR.

oh, i did check the playback speed, but didn't zero in on that flag. thanks for raising the problem! i think we can fix it here whether it's new or not

**Comment by ryan953** [comment]
@c298lee I think it's better now; at least i'm having trouble recreating it when only one window/tab is open. If you have sentry.io open in multiple windows then there is a way to get it to break again.

**Pull request merged**